composer.json

```diff
@@ -20,8 +20,7 @@
         "cakephp/cakephp": "~3.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "4.1.*",
-        "cakephp/cakephp-codesniffer": "dev-master"
```

Comment:
^ what @jadb says. It's installed locally via composer cache, so I don't see the big deal with having it here. Also, I *don't* install anything globally if I can manage it - down that way leads failure...